api/routers/repository.py

```
@@ -163,26 +164,28 @@ async def delete_repository_by_name(
-        if not overview:
-            raise HTTPException(status_code=404, detail="Repository not found")
-
         deleted = query_service.delete_repository(repo_id)
 
+        # Parse owner/repo from repo_id and clean up Firestore
+        if "/" in repo_id:
+            owner, repo_name = repo_id.split("/", 1)
+            try:
+                firebase_service.delete_repo_metadata(user["uid"], owner, repo_name)
+            except Exception:
+                pass  # Non-fatal
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)